src/main/java/uk/gov/ch/controller/update/OverseasEntityBeneficialOwnerController.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.ch.controller.update;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.ch.OracleQueryApplication;
+import uk.gov.ch.exception.BeneficialOwnerCountNotFoundException;
+import uk.gov.ch.model.update.OverseasEntityBeneficialOwner;
+import uk.gov.ch.service.update.OverseasEntityBeneficialOwnerService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.DataMap;
+
+import java.util.List;
+
+@RestController
+public class OverseasEntityBeneficialOwnerController {
+
+    @Autowired
+    private OverseasEntityBeneficialOwnerService overseasEntityBeneficialOwnerService;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OracleQueryApplication.APPLICATION_NAME_SPACE);
+
+    @GetMapping("/overseas-entity/{companyNumber}/beneficial-owners")
+    public ResponseEntity<List<OverseasEntityBeneficialOwner>> getOverseasEntityBeneficialOwners(@PathVariable String companyNumber) {
+        DataMap dataMap = new DataMap.Builder().build();
```

Comment:
Idea behind the new structured logger builder class is that you would call `.companyNumber(companyNumber)` before `.build()` which thereby ensures that you provide a value of the correct type for company number (and all the other fields).

Doing this via a `.put` to the log map on line 30 is actively not benefiting from the new approach. Could you change please and also in the other classes below. Thanks.

And apologies, I may have missed this on previous reviews, but all instances of using the new builder should be changed to work like this.